modules/picard/sortsam/meta.yml

```diff
@@ -0,0 +1,46 @@
+name: picard_sortsam
+description: Sorts BAM/SAM files based on a variety of picard specific criteria
+keywords:
+    - sort
+    - bam
+    - sam
+tools:
+    - picard:
+        description: |
+            A set of command line tools (in Java) for manipulating high-throughput sequencing (HTS)
+            data and formats such as SAM/BAM/CRAM and VCF.
+        homepage: https://broadinstitute.github.io/picard/
+        documentation: https://broadinstitute.github.io/picard/
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - bam:
+      type: file
+      description: BAM/CRAM/SAM file
+      pattern: "*.{bam,sam}"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - version:
+      type: file
+      description: File containing software version
+      pattern: "*.{version.txt}"
+  - bam:
+      type: file
+      description: Sorted BAM/CRAM/SAM file
+      pattern: "*sorted.{bam}"
+  - sort_order:
+      type: value
+      description: Picard sort order type
+      pattern: "unsorted|queryname|coordinate|duplicate|unknown"
```

Comment:
This should be moved to the `input` section

Reply:
Already noticed :sweat_smile: